src/services/services.ts

```diff
@@ -4913,6 +4913,24 @@ namespace ts {
 
             if (!documentation) {
                 documentation = symbol.getDocumentationComment();
+                if ((!documentation || documentation.length === 0) && symbol.flags & SymbolFlags.Property) {

```

Comment:
can we enforce that `getDocumentationComment` always returns an array so we can get rid of extra check?